r1oga/zkitter-ui
# feat: display list of follower/following(s)

Closes #4 #5 
Depends on https://github.com/r1oga/zkitterd/pull/7

[Demo](https://www.loom.com/share/fe524643c22e49a9ae854e62280a9b83)

## Test plan
- follow a user
- your following count should increase
- click on it
- followed user is displayed
- click on followed user
- you goto that user
- that user has an increased count of follower
- click on their follower count
- it displays at least your user
